android/app/src/main/kotlin/app/revanced/manager/flutter/MainActivity.kt

```diff
@@ -27,6 +27,8 @@ private const val INSTALLER_CHANNEL = "app.revanced.manager.flutter/installer"
 class MainActivity : FlutterActivity() {
     private val handler = Handler(Looper.getMainLooper())
     private lateinit var installerChannel: MethodChannel
+    private var run: Boolean = true
```

Comment:
```suggestion
    private var cancel: Boolean = true
```

Maybe rename this to cancel so the if statements are more straightforward with their purpose?